#
# Submitted-By: Sascha Hauer, 2006-03-14
# Committed-By: Sascha Hauer
#
# Error:
# we need this for fb on pxa27x processors
#
# State:
#
# not sent upstream yet
#
Index: include/lcd.h
===================================================================
--- include/lcd.h.orig
+++ include/lcd.h
@@ -77,7 +77,7 @@ typedef struct vidinfo {
 
 extern vidinfo_t panel_info;
 
-#elif defined CONFIG_PXA250
+#elif (defined CONFIG_PXA250 || defined CONFIG_PXA27X)
 /*
  * PXA LCD DMA descriptor
  */
